ipi_tests/regression_tests/runstools.py

```diff
@@ -0,0 +1,114 @@
+import subprocess as sp
+import sys
+from pathlib import Path
+import pytest
+import numpy as np
+import time
+import glob
+import os
+import shutil
+from tempfile import TemporaryDirectory
+import tempfile
+
+
+def get_info_test(parent):
```

Comment:
Comments are missing in this file. Again it becomes tougher to follow without docstrings.